src/js/http_outgoing.js

```diff
@@ -133,15 +141,33 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
     return true;
   }
 
-  var ret = this._send(chunk, encoding, callback);
+  var len, ret;
+  if (this.chunkedEncoding) {
+    if (typeof chunk === 'string') {
+      len = Buffer.byteLength(chunk, encoding);
+    } else {
+      len = chunk.length;
+    }
 
-  return ret;
+    this._send(len.toString(16), 'latin1', null);
```

Comment:
We might have no "latin1" encoding yet.

Reply:
An alias of `latin1` is `binary`, which is one-byte encoded. 
I've tested over this, data have been correctly encoded.

Reply:
Nope, see the following function: https://github.com/Rokid/ShadowNode/blob/3d408ea40191c693e470a9d0a48be60a98b84f38/src/js/net.js#L160-L165

It shows the encoding `latin1` could not be used.

Reply:
looks like it doesn't support specify encoding anyway. but for better compatibility in the future implementations, it's reasonable to leave as it is.

Reply:
The `write(chunk, callback)` only has 2 arguments, however we pass it with 3 arguments. Therefore this makes confusion and inconsistent, let's do that after we changed the function is 100% same as Node.js.

Reply:
Maybe we support encoding in `.write()` method before this?

Reply:
😅Actually the chunk header is a hex number so that's ok to use default ascii encoding. 
It's ok to remove the encoding argument and use default ascii encoding at this scenario.